### PR TITLE
configure service1.bb.net

### PR DESCRIPTION
**Pull request by sa2ajj**

### CODE REVIEW

**Comment by sa2ajj** [comment]
These should be no-op when run on service1.bb.net

File: host-service1.yml [hunk 1]
```diff
@@ -5,5 +5,23 @@
   connection: local
   sudo: yes
   roles:
-  - base-servicehost
```
Comment (djmitche) [inline host-service1.yml]: Why is this removed?

Comment (sa2ajj) [inline host-service1.yml]: jail -> base-jailhost -> base-servicehost
